shell/assets/translations/en-us.yaml

```diff
@@ -3551,6 +3551,7 @@ prefs:
     emacs: 'Emacs'
     vim: 'Vim'
   advanced: Advanced
+  advancedTooltip: Enables 'View in API' option, enables a 'none' locale option, <br />enables shortcuts to toggle 'none' locale (shift+l) and toggle light or dark theme (shift+t)
```

Comment:
Can we remove the information about the none locale - this is internal developer that users don't need to know. We should probably call out the 'View in API' as a specific setting at some point.

Reply:
See https://github.com/rancher/dashboard/pull/6229. `advancedTooltip` text needs updating